<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1417</width>
    <height>858</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Inter</family>
    <pointsize>7</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>인스타 DM 발송_5.0_창 오픈 방식 </string>
  </property>
  <property name="toolTip">
   <string extracomment="4.5버전 - 인터넷 사용기록(쿠키, 캐시 값) 지우는 기능 추가 - 생일 인증 통과 가능"/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #303030;
border-color: #0096C7;
color:white;
QMainWindow{
   background-color: white
}
QFrame { 
   border: 5px solid black 
} 
This worke</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>260</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK KR Medium</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">    background-color: #EBECF1;
	color:black;</string>
    </property>
    <property name="text">
     <string>고객정보</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>460</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK KR Medium</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">    background-color: #EBECF1;
	color:black;</string>
    </property>
    <property name="text">
     <string>발송이력</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>690</y>
      <width>1271</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK KR Medium</family>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string extracomment="4.5 인터넷 사용기록 삭제 기능 추가"/>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color: #2E292D;
border-radius : 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK KR Medium</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">    background-color: #EBECF1;
	color:black;</string>
    </property>
    <property name="text">
     <string>로그인계정정보</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>170</y>
      <width>225</width>
      <height>37</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK KR Black</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>DM 내용(1~4중 랜덤)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_3">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>390</y>
      <width>151</width>
      <height>181</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK KR Medium</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #dbdbdb;
    background-color: #fff;
	color:black;
	border-radius : 10px;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_4">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>390</y>
      <width>151</width>
      <height>181</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK KR Medium</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #dbdbdb;
    background-color: #fff;
	color:black;
	border-radius : 10px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>590</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK KR Black</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #3C64FF;
    border-color: #35c5f0;
    color: #fff;
padding: 13px 10px 14px;    
    line-height: 26px;
border-radius : 10px;
QPushButton#searchButton:hover {
    border-color: #09addb;
    background-color: #09addb;
}  </string>
    </property>
    <property name="text">
     <string>DM발송 시작</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>70</y>
      <width>151</width>
      <height>37</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK KR Black</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;멀티 키워드를 지원합니다. 입력값은 ,(콤마)로 구분합니다. (띄어쓰기 없이)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>키워드(콤마로 구분)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>110</y>
      <width>261</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK KR Black</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #dbdbdb;
    background-color: #fff;
	color:black;
	border-radius : 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>220</y>
      <width>151</width>
      <height>161</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK KR Medium</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #dbdbdb;
    background-color: #fff;
	color:black;
	border-radius : 10px;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>220</y>
      <width>151</width>
      <height>161</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK KR Medium</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #dbdbdb;
    background-color: #fff;
	color:black;
	border-radius : 10px;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>1041</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #dbdbdb;
    background-color: #fff;
	color:black;
	border-radius : 10px;</string>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragOnly</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>1041</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #dbdbdb;
    background-color: #fff;
	color:black;
	border-radius : 10px;</string>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragOnly</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>1041</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #dbdbdb;
    background-color: #fff;
	color:black;
	border-radius : 10px;</string>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragOnly</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>590</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK KR Black</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;DB 정보를 갱신해서 다시 불러옵니다.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #383838;
    border-color: #35c5f0;
    color: #fff;
padding: 13px 10px 14px;    
    line-height: 26px;
border-radius : 10px;
QPushButton#searchButton:hover {
    border-color: #09addb;
    background-color: #2192FF;
	border-radius : 10px;
}</string>
    </property>
    <property name="text">
     <string>새로고침</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>1071</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #EBECF1;
border-radius : 10px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>580</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2E292D;
border-radius : 10px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>frame_2</zorder>
   <zorder>frame</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>textBrowser</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_9</zorder>
   <zorder>textEdit_3</zorder>
   <zorder>textEdit_4</zorder>
   <zorder>pushButton</zorder>
   <zorder>label_2</zorder>
   <zorder>lineEdit</zorder>
   <zorder>textEdit_2</zorder>
   <zorder>textEdit</zorder>
   <zorder>tableWidget</zorder>
   <zorder>tableWidget2</zorder>
   <zorder>tableWidget3</zorder>
   <zorder>pushButton_2</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
